server/server.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const db = require('../database/database.js')
+const mongoose = require('mongoose');
+const utils = require('./utils.js')
+
+const app = express();
+const port = 3001;
+
+app.use(express.static(__dirname + '/../client/dist'));
+app.use(bodyParser.urlencoded({extended: true}))
+
+app.get('/', (req, res) => {
```

Comment:
you could probably get rid of this route unless you're planning on saving it for later